x-pack/packetbeat/scripts/mage/pcap.go

```diff
@@ -0,0 +1,87 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package mage
+
+import (
+	"errors"
+	"fmt"
+	"io/fs"
+	"os"
+	"path/filepath"
+	"runtime"
+	"strings"
+
+	"github.com/magefile/mage/sh"
+
+	devtools "github.com/elastic/beats/v7/dev-tools/mage"
+)
+
+// NpcapVersion specifies the version of the OEM Npcap installer to bundle with
+// the packetbeat executable. It is used to specify which npcap builder crossbuild
+// image to use and the installer to obtain from the cloud store for testing.
+const (
+	NpcapVersion = "1.80"
+	installer    = "npcap-" + NpcapVersion + "-oem.exe"
+)
+
+func ImageSelector(platform string) (string, error) {
+	image, err := devtools.CrossBuildImage(platform)
+	if err != nil {
+		return "", err
+	}
+	if os.Getenv("CI") != "true" && os.Getenv("NPCAP_LOCAL") != "true" {
+		return image, nil
+	}
+	if platform == "windows/amd64" {
+		image = strings.ReplaceAll(image, "beats-dev", "observability-ci") // Temporarily work around naming of npcap image.
+		image = strings.ReplaceAll(image, "main", "npcap-"+NpcapVersion+"-debian9")
+	}
+	return image, nil
+}
+func CopyNCAPInstaller() error {
```

Comment:
`CopyNCAPInstaller` -> `CopyNPCAPInstaller`

And insert a newline in between the functions please.